beancount_chase/checking.py

```diff
@@ -8,7 +8,7 @@
 from beancount.core import data
 from beancount.core import flags
 from beancount.core import number as beancount_number
-from beangulp import importer
+from beangulp.importer import Importer
```

Comment:
Can we stick to the [convention of importing modules rather than classes](https://google.github.io/styleguide/pyguide.html#22-imports)?

So we'd keep this as `from beangulp import importer` and refer to `Importer` as `importer.Importer`. 

(ditto throughout)